src/traces/sankey/render.js

```diff
@@ -79,7 +79,7 @@ function sankeyModel(layout, d, traceIndex) {
             }
         }
 
-        graph.nodes.push({
+        graph.nodes.unshift({
```

Comment:
Cool. Can we add a test to :lock: this down?

Reply:
Same here. We should try :lock: this down.

Reply:
Done in ffbafa7